src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx

```diff
@@ -5476,4 +5476,8 @@ This option should be used when specifying the certificate via -CertificateSubje
   <data name="CommandNoServiceEndpointDescription" xml:space="preserve">
     <value>Does not append "api/v2/packages" to the source URL.</value>
   </data>
+  <data name="SourcesCommandValidAuthenticationTypesDescription" xml:space="preserve">
+    <value>Comma-separated list of valid authentication types for this source, or * to allow all authentication types, which is the default. Example: basic,negotiate</value>
```

Comment:
Don't think we should support `*`, nothing should imply all.

Reply:
How do you specify on the command line that you want to *change* the authentication types from a specific value to all, while also allowing the user to leave the authentication types untouched while updating the username and password?